pkg/cmd/roachtest/test_runner.go

```diff
@@ -2010,3 +2011,12 @@ func grafanaAnnotateTestStart(ctx context.Context, t test.Test, c cluster.Cluste
 		t.L().Printf(errors.Wrap(err, "error adding annotation for test start").Error())
 	}
 }
+
+// logFatalfCtx logs the message using the provided logger and then closes the
+// logger and exits the process with status 1. It should only be used in
+// circumstances where the process cannot continue, and not by tests.
+func logFatalfCtx(ctx context.Context, l *logger.Logger, f string, args ...interface{}) {
+	l.ErrorfCtx(ctx, f, args...)
```

Comment:
Shouldn't we be passing the depth argument to skip the `logFatalfCtx` call frame?

Reply:
Mmm, good point, I'll add it.